pootle/apps/pootle_translationproject/models.py

```diff
@@ -323,6 +298,38 @@ def is_accessible_by(self, user):
 
         return self.project.code in Project.accessible_by_user(user)
 
+    def can_be_inited_from_templates(self):
+        """Returns `True` if the current translation project hasn't been
+        saved yet and can be initialized from templates.
+        """
+
+        if self.id is None:
```

Comment:
can we move this condition back to save - i think this function should work irrespective of whether pk has been set